geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-api/src/main/scala/org/locationtech/geomesa/fs/storage/api/StorageMetadata.scala

```diff
@@ -145,16 +145,20 @@ object StorageMetadata {
   }
 
   /**
-    * Holds a storage file
-    *
-    * @param name file name (relative to the root path)
-    * @param timestamp timestamp for the file
-    * @param action type of file (append, modify, delete)
-    */
+   * Holds a storage file
+   *
+   * @param name file name (relative to the root path)
+   * @param timestamp timestamp for the file
+   * @param action type of file (append, modify, delete)
+   * @param sort sort fields, if any, as feature type attribute number
+   * @param bounds known bounds, if any, keyed by feature type attribute number
+   */
   case class StorageFile(
       name: String,
       timestamp: Long,
-      action: StorageFileAction.StorageFileAction = StorageFileAction.Append
+      action: StorageFileAction.StorageFileAction = StorageFileAction.Append,
+      sort: Seq[Int] = Seq.empty,
+      bounds: Seq[(Int, String, String)] = Seq.empty
```

Comment:
Does this change the format for the metadata?  

If so, any migration or compatibility considerations?

Reply:
Assuming this changes the stored metadata, are there unit tests covering the new data?

Reply:
it does, but the readers can handle both formats. if you write data with a newer version, you won't be able to read it back out. currently these are unused, but i wanted to add them as placeholders. there are unit tests checking the old schemas that cover this.

Reply:
Do we need to mention that in the upgrade guide?  (E.g. not to mix and match new and old writers?)

Reply:
oh, we actually already have it in there due to the key-value metadata change :D